5calls/app/src/main/java/org/a5calls/android/a5calls/model/AccountManager.java

```diff
@@ -0,0 +1,94 @@
+package org.a5calls.android.a5calls.model;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import org.a5calls.android.a5calls.FiveCallsApplication;
+
+/**
+ * A singleton responsible for managing all local account-related information
+ */
+public enum AccountManager {
+    Instance;
```

Comment:
Would it make sense to put this in the AppSingleton instead of using an Instance? I'm not sure which is better.